plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2605,7 +2605,7 @@ public void createView(ConnectorSession session, SchemaTableName viewName, Conne
         Optional<Table> existing = metastore.getTable(viewName.getSchemaName(), viewName.getTableName());
         if (existing.isPresent()) {
             if (!replace || !isPrestoView(existing.get())) {
-                throw new ViewAlreadyExistsException(viewName);
```

Comment:
There are other uses of `ViewAlreadyExistsException` which may be probably replaced by `TableAlreadyExistsException`.

If you replace all of them, then probably `ViewAlreadyExistsException` may be removed.

cc @electrum

Reply:
`ViewAlreadyExistsException` is required for the case when a view already exists while creating a view.

This PR specially deals with when A table exists while creating a view.

Reply:
In the cases where currently `ViewAlreadyExistsException` we are doing previously `metastore.getTable(..)` calls.
I'm thinking that if the metastore calls for a "table", then `TableAlreadyExistsException` is appropriate.